src/app/core/services/models/contract-offer-dto.ts

```diff
@@ -9,5 +9,5 @@ export interface ContractOfferDto {
   policy: Policy;
   provider: string;
   consumer: string;
-  asset: AssetDto;
+  asset: UiAsset;
 }
```

Comment:
this entire class is not very useful, so if this PR also migrates the catalog, this class should disappear